File: demo/MoffatBackend/src/main/java/com/csd460/moffat/model/room/Room.java
package com.csd460.moffat.model.room;

public class Room {
    public long roomId;
    public String bedType;
    public int numberOfBeds;
    public int maxGuests;
    public double price;

    public Room(long roomId, String bedType, int numberOfBeds, int maxGuests, double price) {
        this.roomId = roomId;
        this.bedType = bedType;
        this.numberOfBeds = numberOfBeds;
        this.maxGuests = maxGuests;
        this.price = price;
    }
    public Room(String bedType, int numberOfBeds, int maxGuests, double price) {
        this.bedType = bedType;
        this.numberOfBeds = numberOfBeds;
        this.maxGuests = maxGuests;
        this.price = price;
    }

    public long getId() {
        return roomId;
    }

    public void setId(long roomId) {
        this.roomId = roomId;
    }

    public String getBedType() {
        return bedType;
    }

    public void setBedType(String bedType) {
        this.bedType = bedType;
    }

    public int getNumberOfBeds() {
        return numberOfBeds;
    }

    public void setNumberOfBeds(int numberOfBeds) {
        this.numberOfBeds = numberOfBeds;
    }

    public int getMaxGuests() {
        return maxGuests;
    }

    public void setMaxGuests(int maxGuests) {
        this.maxGuests = maxGuests;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    @Override
    public String toString() {
        return "Room roomId=" +roomId + ", bedType=" + bedType + ", numberOfBeds=" + numberOfBeds + ", maxGuests=" + maxGuests
                + ", price=" + price + "]";
    }
}

File: src/frontend/public/model/room.php
<?php

class Room {
    public $room_id;
    public $bed_type;
    public $number_of_beds;
    public $max_guests;
    public $price;

    public function __construct($room_id, $bed_type, $number_of_beds, $max_guests, $price){
        $this->room_id = $room_id;
        $this->bed_type = $bed_type;
        $this->number_of_beds = $number_of_beds;
        $this->max_guests = $max_guests;
        $this->price = $price;
    }

    public function get_room_id(){
        return $this->room_id;
    }

    public function get_bed_type(){
        return $this->bed_type;
    }

    public function get_number_of_beds(){
        return $this->number_of_beds;
    }

    public function get_max_guests(){
        return $this->max_guests;
    }

    public function get_price(){
        return $this->price;
    }
}

?>
